lib/tenstorrent/bh_arc/init_common.c

```diff
@@ -84,14 +93,16 @@ void DeassertTileResets(void)
 
 int InitFW(uint32_t app_version)
 {
+	int ret;
+
 	WriteReg(STATUS_FW_VERSION_REG_ADDR, app_version);
 
 	/* Initialize ARC DMA */
 	ArcDmaConfig();
 	ArcDmaInitCh(0, 0, 15);
 
 	/* Initialize SPI EEPROM and the filesystem */
-	InitSpiFS();
+	ret = InitSpiFS();
```

Comment:
Probably should add some error handling here.

A `LOG_ERR()` closest to the error point would be good too, if it doesn't exist already

Reply:
Added a LOG_ERR() and returned status code